README.md

```diff
@@ -127,7 +127,7 @@ simple - adapt as required for your distribution:
         )
    ```
    
-   This configuration allows easy switching between server and desktop modes
+   This configuration allows for easy switching between server and desktop modes
```

Comment:
That sounds like American phrasing rather than English (the country from which pgAdmin originates, not the language).

Reply:
Thanks a lot, Mr. Dave :))